library/Rules/Date.php

```diff
@@ -49,8 +49,39 @@ public function validate($input)
             $this->format = $exceptionalFormats[$this->format];
         }
 
-        $info = date_parse_from_format($this->format, $inputString);
+        return $this->isValidDateTime($input);
+    }
+
+    private function isValidDateTime($input)
+    {
+        $info = date_parse_from_format($this->format, $input);
+        if (!$this->isParsable($info)) {
+            return false;
+        }
+
+        if (!$this->hasDateFormat()) {
```

Comment:
If it doesn't have a Date format, is it `true`?

Reply:
It means that I don't need to validate the date

Reply:
I changed the order of the conditions to make it more clear